x/stake/keeper/inflation.go

```diff
@@ -40,7 +41,7 @@ func (k Keeper) nextInflation(ctx sdk.Context) (inflation sdk.Rat) {
 	// 7% and 20%.
 
 	// (1 - bondedRatio/GoalBonded) * InflationRateChange
-	inflationRateChangePerYear := sdk.OneRat().Sub(pool.bondedRatio().Quo(params.GoalBonded)).Mul(params.InflationRateChange)
+	inflationRateChangePerYear := sdk.OneRat().Sub(pool.BondedRatio().Quo(params.GoalBonded)).Mul(params.InflationRateChange)
```

Comment:
I think we need to handle the case where params.GoalBonded is 0 here. Otherwise this causes a zero division error in certain circumstances. (#1269)

Reply:
See my comment in the issue, I think the better solution is to handle this further upstream and throw an error at initialization if the goal bonded = 0. aka. going to this code as is, but also fix basecoin / add some other upstream checks